manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -131,6 +131,12 @@ rules:
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["create", "get", "list", "watch", "update"]
+  - apiGroups: ["admissionregistration.k8s.io"]
```

Comment:
Note - this role is called `istio-reader` but it essentially gives root control. This seems a bit surprising. It is blocked by `centralIstiod` though so maybe not a huge issue

Reply:
yes, there is a https://github.com/istio/istio/issues/23263 to clean this